spec/models/eligibility_check_spec.rb

```diff
@@ -289,15 +289,15 @@
         { completed_requirements: true, region: create(:region) }
       end
 
-      it { is_expected.to eq(:degree) }
+      it { is_expected.to eq(:qualification) }
     end
 
     context "when degree is present" do
       let(:attributes) do
         { degree: true, completed_requirements: true, region: create(:region) }
```

Comment:
```suggestion
        { qualification: true, completed_requirements: true, region: create(:region) }
```

As the next question is the qualification here we should change the test case, and then assert that the status becomes `degree`.

Reply:
Otherwise we've got two tests that assert for `teach_children`.